### DISCUSSION TITLE
Fast-spectrum MSR discussion

### DISCUSSION THREAD

**Discussion by nukeman4444** [arfc/moltres#262] [General / Ideas]
### Discussed in https://github.com/arfc/moltres/discussions/195

<div type='discussions-op-text'>

<sup>Originally posted by **smpark7** April  5, 2022</sup>

## 👋 Welcome!
  We’re exploring replacing our [moltres-users](https://groups.google.com/g/moltres-users) Google Group with Discussions as a place to connect with other members of our community. We hope that the increased accessibility/visibility will encourage you to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

  The Google Group will remain as an archive of past content. Please create all new posts on this Discussions platform.

### Tips
- Use the Discussions categories to place your new posts in the appropriate category. The `Labels` options currently apply to issues and PRs only.
- You may link issues and PRs here using the same `#123` format and vice versa.
- Copy-paste your code instead of sharing a screenshot. Pasting text helps other users find your post using the search function when they encounter similar issues. It will also help us parse through your code quicker.
- For large text files, you may upload the file directly. Please refrain from posting huge binary files.</div>

**Comment by nukeman4444** [comment]
I am interested in having a conversation with anyone that may have worked on a MOOSE model for the Molten Cloride Reactor Experiment (MCRE) being constructed at INL.  See INL/RPT-22-68976.  Also anyone that may be working on a followup demonstration concept for Fast Reactors using an eutectic mixture of sodium chloride (NaCl) and uranium trichloride (UCl3).

**Comment by smpark7** [comment]
Hi @nukeman4444. We do have a moltres user currently working on a Moltres model for MCRE. Can I ask why you're interested in that work?

**Comment by nukeman4444** [comment]
I'm a retired nuclear engineer. I don't understand why there isn't more emphasis on reactors that can use our spent nuclear fuel to produce electricity.    Molten salt fast reactors have the potential to make that happen.    I probably am not going to contribute much but I have a Grandson that is interested.   
  

  

  

  
  
  
  
  
>

**Comment by nukeman4444** [comment]
Is anyone working on a follow-on power demonstration project for this concept.  By that I mean one that adds heat exchangers and turbines to the model?  This would allow one to check out the feasibility of having one reactor vessel of a given fixed size power a wide range of heat exchange and turbine size by varying the flow rate and enrichment in the fixed size vessel.

**Comment by nukeman4444** [comment]
Here is a link to the Elysium reactor concept I mentioned in the message above.
https://www.intelligentliving.co/wp-content/uploads/2020/11/ELYSIUM-TECHNOLOGIES-USA-585x328.jpg
I have been following the nuclear power industry since I graduated in1964.  I like this concept as it has a free or maybe even a negative cost if fuel as it can burn up the tonnes of Uranium and Plutonium waste.  Since it operates at near atmospheric pressure it avoids all the cost and safety issues involved with our existing high pressure machines.  Could your MOOSE/Moltres model be used to verify this is a feasible concept?

**Comment by smpark7** [comment]
I modeled a preliminary design of the European molten salt fast reactor (MSFR) for my Master's work. You may find it at this [link](https://www.ideals.illinois.edu/items/116168). Another grad student is currently working on a Moltres model for the MCRE, but it is currently a work in progress.

I'm not an expert on policy but I believe some of the obstacles for using spent fuel are establishing safety and licensing processes for commercial fast reactors, reprocessing costs vs cheap uranium, and the federal ban on fuel reprocessing. Sodium fast reactors have had their own technical and policy-related issues in the past decades.

TerraPower, Kairos Power, and Terrestrial Energy are currently the leading nuclear companies regarding plans for demonstration and licensing of MSRs. Other new reactor concepts which are probably closer to deployment are water-cooled small modular reactors, heat-pipe microreactors, and high temp gas-cooled (micro)reactors. You may refer to the DOE's [Advanced Reactor Demonstration Program](https://www.energy.gov/ne/advanced-reactor-demonstration-program).

Moltres currently uses approximations for the heat exchanger and coolant pump. You'll probably find better models for those parts in other purpose-built MOOSE apps like the SAM systems code. We're currently better equipped to model 2D or 3D neutronics and thermal hydraulics phenomena in the reactor core. This may change since Moltres is under active development.

Creating a Moltres model for the Elysium reactor is definitely feasible. It would need to be done with another neutronics code for generating group constant data that Moltres requires. However, you'll notice that many private companies do not have their exact designs publicly available.

**Comment by nukeman4444** [comment]
To smpark7:
I was very impressed by your thesis.  Thanks for sending the link.  I now have a much better feel of how MOOSE can be applied to practical problems.  I have a Masters Degree in Nuclear Engineering that I received in June 1964.  Wow, it looks like that is 60 years ago.  What is the status of the European MSFR?  Are they building a low power demonstration plant?  I seems like stretch to go directly with a 1300 MWe machine?

**Comment by smpark7** [comment]
I appreciate your kind words. The European MSFR is a conceptual design and a result of EU-wide collaboration (universities and research institutes) on MSR development with EU funding. Their focus was on experimental proof of concept and safety assessment of the fast-spectrum MSR concept. Many researchers and grad students involved in the project have formed or joined European MSR startups (e.g., Copenhagen Atomics, Thorizon, Seaborg Technologies). They're aiming for smaller designs which are modular or floating concepts to gain competitive edges over other designs/types.

P.S. I'm editing the title and moving this post to the discussion section

**Comment by nukeman4444** [comment]
Someone asked why I was interested in this discussion.  When I graduated in 1964 the US was on its way to building about 100 large nuclear plants.  Now these plants are being shutdown  at an increasing rate.  I am not sure we have build more than a couple plants in the last 20 years.  With many coal plants also being shutdown we are going to have an urgent need for more base-load power.  I think Molten Salt Fast Reactors are the way to go.  They get rid of the expense of containing the energy associated with a high pressure machine.  They also eliminate the risks associated with of the issue of excess reactivity.  The Fast reactor also has the ability to use the vast amount of spent fuel, and excess weapons grade fissile material we have built up over the last 70 years to produce clean energy.  If one includes the amount of depleted uranium there is an estimate that the US can be powered at the current demand for hundreds of years.  The reason I like the Elysium Sodium Chloride Fast Reactor Concept is because it has a higher energy spectrum,  has a more reasonable melting and boiling point, and it has less proliferation issues.  I think Moltres could be used to verify the validity of this concept.  The initial reactors would probably have to be built at a site where higher enriched uranium and plutonium are available like INL or Savannah River.

**Reply by nukeman4444** [reply]
Is anyone in the world actively working on a Power Demonstration Sodium Chloride Fast Reactor?  In the US Elysium talked about this in their videos but I don't think they are pursuing it any more.    TerraPower and Southern Company are working on this technology as shown in the following link: https://www.terrapower.com/our-work/molten-chloride-fast-reactor-technology/  This article says the government as put some $40 million into this project.  They have a picture of what looks like a power reactor but no details.  A lot of work seems to be going on in Everett, Washington and on the Molten Chloride Reactor Experiment (MCRE) Project at INL.  With all this money and activity going on isn't Moltres receiving funding to support these efforts?

**Reply by smpark7** [reply]
I have not heard much about Elysium recently. TerraPower and Southern Company are currently receiving the most funding support for a fast MSR concept. Kairos Power received significant funding as well but they're working on a fluoride salt-cooled high-temperature reactor which has a prismatic core with TRISO fuel. Overall, we still have many steps to take before deploying the first of a kind commercial fast MSR. Several codes exist for modeling MSRs. Moltres is one of the few that are open source. TerraPower have their own in-house codes for MSRs.

That funding mostly goes to TerraPower, Southern Company, and their national lab collaborators. Much of their funding also goes into experimental setups and the actual construction of the MCRE. For us, as a university research group, our funding usually comes from other sources, such as the NEUP and other programs directed at supporting universities.

Many other companies and institutes are working on other technological hurdles arising from using molten salt. We're still missing key experimental data about molten salt properties, interactions with structural components, salt reprocessing techniques at high temperatures.
